packages/web/pages/stake/index.tsx

```diff
@@ -206,10 +208,14 @@ export const Staking: React.FC = observer(() => {
       return;
     }
 
-    // TODO add showValidatorNextStepModal here
+    const selectedKeepValidators = localStorage.getItem("keepValidators");
```

Comment:
Decided to use localhost because our use case is simple enough not to require indexedDb. in the future, we will look into storing this on chain with the `address` as a key.

Reply:
https://github.com/osmosis-labs/osmosis-frontend/blob/581b32f4fe78e0fe1abf7ea70f86e3feea27f1fd/packages/web/hooks/window/use-localstorage-state.ts#L7

Heads up we do have this `useLocalstorageState` hook that may simplify calling code